requirements.txt

```diff
@@ -5,7 +5,7 @@ astropy==5.3.4
 astroquery==0.4.6
 beautifulsoup4==4.12.2
 fitsio==0.9.12
-flaky==3.7.0
+flaky==3.8.1
```

Comment:
Seems if it's flaky, we ought to randomly install a different version every time....

Reply:
:)